.github/workflows/upgradebot.yml

```diff
@@ -0,0 +1,27 @@
+name: Upgrade geth bot
+on:
+  push:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 6,12 * * *'
```

Comment:
I would shift the 6 to 4, to be earlier also for.

do we need the 12? not sure if we need more than daily.